Content.Server/Botany/Components/ThornyComponent.cs

```diff
@@ -0,0 +1,16 @@
+using Robust.Shared.Containers;
+using Content.Shared.Damage;
+
+namespace Content.Server.Botany.Components;
+[RegisterComponent]
+public sealed partial class ThornyComponent : Component
+{
+    [DataField]
+    public string Sound = string.Empty;
+
+    [DataField]
+    public int ThrowStrength = 3;
```

Comment:
Rename this one into ThrowSpeed, so its meaning is more clear. The same thing was recently done for the TryThrow function parameter.